src/Models/Product.php

```diff
@@ -20,10 +23,70 @@ protected static function booting()
         static::addGlobalScope(function (Builder $builder) {
             $builder->whereIn('visibility', config('rapidez.statamic.runway.product_visibility'));
         });
+
+        static::updating(function (Product $product) {
+            $productEntry = $product->entry;
+
+            if (!$productEntry) {
+                $productEntry = Entry::make()
+                    ->collection('products')
+                    ->locale(Site::selected()->handle())
+                    ->data(['linked_product' => $product->sku]);
```

Comment:
Currently we link through `entity_id` , i think this needs to be kept in order to support backwards compatibility with content already configured in the old situation.

Reply:
Nope, for products the SKU is used. Categories use the `entity_id`.